src/s3/util/fileViewerManager.ts

```diff
@@ -112,20 +111,24 @@ export class S3FileViewerManager {
                 return undefined
             }
 
-            getLogger().debug(`user confirmed download, continuing`)
-            showOutputMessage(`user confirmed download, continuing`, this.outputChannel) //TODOD:: debug log,
+            getLogger().debug(`FileViewer: User confirmed download, continuing`)
+            showOutputMessage(
```

Comment:
This one probably isn't necessary for the user. Maybe have a message on cancellation, but not continue?